docs/en/tutorials/optim_wrapper.md

```diff
@@ -1,3 +1,500 @@
+
+Now we take setting up a SGD OptimWrapper as an example.
+
+```python
+optimizer = dict(type='SGD', lr=0.01, momentum=0.9, weight_decay=0.0001)
+optim_wrapper = dict(type='OptimWrapper', optimizer=optimizer)
+```
+
+Here we configured a OptimWrapper with the type of SGD, and other parameters such as learning rate, momentum, etc. Considerting that OptimWrapper is standard single-precision training, we can skip the `type` as follows.
+
+```python
+optimizer = dict(type='SGD', lr=0.01, momentum=0.9, weight_decay=0.0001)
+optim_wrapper = dict(optimizer=optimizer)
+```
+
+To enable mixed-precision training and gradient accumulation, we change `type` to `AmpOptimWrapper` and specify the `accumulative_counts` parameter.
+
+```python
+optimizer = dict(type='SGD', lr=0.01, momentum=0.9, weight_decay=0.0001)
+optim_wrapper = dict(type='AmpOptimWrapper', optimizer=optimizer, accumulative_counts=2)
+```
+
+```{note}
+If this is your first tutorial document of MMEngine and you are not familiar with concepts like [configs](../advanced_tutorials/config.md), and [registries](../advanced_tutorials/registry.md), it is recommended to skip the following advanced usage instructuions and finish other documents first. Of course, if you have already familiar with these parts, we highly recommend the advanced part which covers:
+
+1. how to customize the learning rate, decay coefficient, and other model parameters of the optimizer in the configuration.
+2. how to customize the construction policy of the optimizer.
+
+Apart from the pre-requisite knowledge of the configs and the registries, we also recommend to start the advanced section with a thorough understanding of the parameter in Pytorch's native optimizer construction.
+```
+
+## Advanced usages
+
+PyTorch's optimizer allows different hyperparameters to be set for each parameter in the model, such as using different learning rates for the backbone and head of a classification model.
+
+```python
+from torch.optim import SGD
+import torch.nn as nn
+
+model = nn.ModuleDict(dict(backbone=nn.Linear(1, 1), head=nn.Linear(1, 1)))
+optimizer = SGD([{'params': model.backbone.parameters()},
+     {'params': model.head.parameters(), 'lr': 1e-3}],
+    lr=0.01,
+    momentum=0.9)
+```
+
+In the above example, we set a learning rate of 0.01 for the backbone, while another learning rate of 1e-3 for the head. Users can pass a dict list of parameters and corresponding hyperparameters of different parts of their model to the optimizer to achieve the fine-grained tuning.
+
+In MMEngine, the optimizer wrapper constructor allows users to set hyperparameters in different parts of the model directly by setting the `paramwise_cfg` in the configuration file rather than by modifying the code.
```

Comment:
```suggestion
In MMEngine, the optimizer wrapper constructor allows users to set hyperparameters in different parts of the model directly by setting the `paramwise_cfg` in the configuration file rather than by modifying the code of building the optimizer.
```